crates/hotshot/Cargo.toml

```diff
@@ -54,7 +54,7 @@ snafu = { workspace = true }
 surf-disco = { workspace = true }
 time = { workspace = true }
 tracing = { workspace = true }
-anyhow = { workspace = true }
+anyhow.workspace = true
```

Comment:
Curious what this change here introduces.

Reply:
Hahaha whoops, this was totally some weird merge conflict behavior. It looks like both the merge and I added `anyhow`

Reply:
Ah yeah that was me. Sorry about that!